src/templates/index.jsx

```diff
@@ -0,0 +1,77 @@
+
```

Comment:
Umwandlung des Index-Page-Elements in ein Template, um die beiden Dateien `pages/index.de.js` und `pages/index.en.js` verwerfen zu können, die doppelten Code aufwiesen.